risk-analysis/src/main/scala/it/pagopa/interop/commons/riskanalysis/model/RiskAnalisys.scala

```diff
@@ -0,0 +1,29 @@
+package it.pagopa.interop.commons.riskanalysis.model
+
+import scala.io.Source
+
+sealed trait RiskAnalysisModel
+
+final case class RiskAnalysisForm(version: String, answers: Map[String, Seq[String]])
+final case class RiskAnalysisMultiAnswerValidated(key: String, values: Seq[String])           extends RiskAnalysisModel
+final case class RiskAnalysisSingleAnswerValidated(key: String, value: Option[String] = None) extends RiskAnalysisModel
+final case class RiskAnalysisFormSeed(
+  version: String,
+  singleAnswers: Seq[RiskAnalysisSingleAnswerValidated],
+  multiAnswers: Seq[RiskAnalysisMultiAnswerValidated]
+) extends RiskAnalysisModel
+
+sealed trait RiskAnalysisTenantKind
+
+object RiskAnalysisTenantKind {
+  case object PA      extends RiskAnalysisTenantKind
+  case object GSP     extends RiskAnalysisTenantKind
+  case object PRIVATE extends RiskAnalysisTenantKind
+}
+
+object RiskAnalysisTemplate {
+  val riskAnalysisTemplate = Source
```

Comment:
Ok, I just saw this through the purpose-process PR.
I was wondering if this should stay here or if it should be kept in purpose-process?
@galales what do you think?
Personally, I will keep it in purpose-process because it's used to generate the PDF, but I don't have other arguments.
(@nttdata-rtorsoli In case of keeping [this](https://github.com/pagopa/interop-commons/pull/283#issuecomment-1729486881) message is no more valid)

Reply:
I agree, this is specific to the creation of the PDF and it may stay in the purpose-process 👍